docs/DMS/PRIMARY-DATA-STORAGE/README.md

```diff
@@ -43,9 +56,17 @@ do it whenever possible.
 We need to make sure that we design to meet performance goals, and test early and often to ensure that we are
 able to hit those goals.
 
-## Implementation
+## Solution
 
-### ER Diagram
+> [!TIP]
+> See [Design Options for Data Management Service Data Storage](./PRIMARY-DATA-STORAGE-alternatives.md)
+> for detailed analysis of potential solutions, including drawbacks and mitigating factors.
+
+### General Structure
+
+This is very much like the Meadowlark implementation of the PostgreSQL backend, except we want to take
+advantage of foreign key constraints for reference validation. We also want to introduce partitioning. All
+three tables use the sequential surrogate key pattern with size `BIGINT`.
```

Comment:
@bradbanister you'll see that I re-arranged your text, but I did not edit any of it. I think the only edit I applied to existing content, other than re-arranging, was to change `VARBINARY` to `VARCHAR`.